packages/Takeover/src/Takeover.js

```diff
@@ -0,0 +1,116 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import PropTypes from "prop-types";
+import { Transition } from "react-transition-group";
+import FocusTrapLibrary from "focus-trap-react";
+import styled from "styled-components";
+import EscListener from "./components/EscListener";
+import Wrapper from "./components/Wrapper";
+import Header from "./components/Header";
+import LockBodyScroll from "./components/LockBodyScroll";
+import { animationDuration } from "./tokens";
+import extractChildren from "./helpers/extractChildren";
+
+const propTypes = {
+  /** The content for the Takeover. */
+  children: PropTypes.node.isRequired,
+
+  /** Control the visibility of the takeover */
+  isOpen: PropTypes.bool.isRequired,
+
+  /** Callback triggered when the takeover needs to be close */
+  onClose: PropTypes.func,
+
+  /** Callback once the takeover has been opened event */
+  onAfterOpen: PropTypes.func,
+
+  /** Callback once the takeover has been closed event */
+  onAfterClose: PropTypes.func,
+
+  /** Render the takeover inline */
+  isInline: PropTypes.bool,
+
+  /** Takeover will capture and will not propagate event after pressing ESC. Helps in case we have many overlapped components. */
+  shouldStopEscapePropagation: PropTypes.bool,
+};
+
+const defaultProps = {
+  isInline: false,
+  shouldStopEscapePropagation: false,
+  onAfterClose: () => {},
+  onClose: () => {},
+  onAfterOpen: () => {},
+};
+
+const Portal = ({ children, active }) =>
+  active ? ReactDOM.createPortal(children, document.body) : <React.Fragment>{children}</React.Fragment>;
+
+const StyledWrapper = styled(Wrapper)`
+  display: flex;
+  flex-direction: column;
+`;
+
+const StyledHeader = styled(Header)`
+  flex: none;
+`;
+
+const ContentWrapper = styled.div`
+  overflow-y: auto;
+  flex: 1 1 auto;
+`;
+
+const Takeover = ({ isOpen, onClose, isInline, shouldStopEscapePropagation, onAfterClose, onAfterOpen, ...props }) => {
```

Comment:
please use a `function Takeover(props)`  instead of declaring a variable, and use destructuring inside of the function, again this is just for the sake of consistency and how other components are doing it on our library.